pagerduty/event_orchestration_path_util.go

```diff
@@ -54,6 +54,47 @@ var eventOrchestrationPathExtractionsSchema = map[string]*schema.Schema{
 	},
 }
 
+var eventOrchestrationAutomationActionObjectSchema = map[string]*schema.Schema{
```

Comment:
The schemas and methods below are shared between Service and Global orchestrations so I moved them here from`pagerduty/resource_pagerduty_event_orchestration_path_service.go`